### DISCUSSION TITLE
Is it possible to configure LazyVim terminal to use the Fish Shell?

### DISCUSSION THREAD

**Discussion by andreacfromtheapp** [LazyVim/LazyVim#3865] [Q&A]
Hi :)

I've recently switched to fish shell but I have left $SHELL as zsh on macOS, just in case.

Is it possible to configure LazyVim terminal to use fish shell instead of the system shell?

Please and thank you

**Comment by dpetka2001** [comment]
Try to set in your `options.lua` file the option `vim.o.shell = "fish"`.

**Reply by andreacfromtheapp** [reply]
> Try to set in your `options.lua` file the option `vim.o.shell = "fish"`.

thank you!
